docs/modules/user-guide/partials/proc_adding-a-registry-schema.adoc

```diff
@@ -0,0 +1,423 @@
+[id="adding-a-registry-schema_{context}"]
+= Adding a registry schema
+
+[role="_abstract"]
+To add samples and stacks from other repositories, put the file, `extraDevfileEntries.yaml`, under the root of a devfile registry. This file contains the location information of the extra samples and stacks.
```

Comment:
The opening sentence is a little awkward with all of the commas. Let's rewrite just a little bit to make it flow more naturally:

```adoc
To add samples and stacks from other repositories, place the `extraDevfileEntries.yaml` file under the root of a devfile registry.
```